package.json

```diff
@@ -18,7 +18,8 @@
     "lint": "eslint --max-warnings 0 --cache --ext .jsx,js packages",
     "test-watch": "TEST_WATCH=1 meteor test-packages ./packages/lesswrong --driver-package meteortesting:mocha --settings sample_settings.json --port 3200 --full-app",
     "test": "meteor test-packages ./packages/lesswrong --driver-package meteortesting:mocha --settings sample_settings.json --port 3200 --full-app --once",
-    "test-prod": "meteor test-packages ./packages/lesswrong --driver-package meteortesting:mocha --settings settings.json --port 3200 --full-app --once"
+    "test-prod": "meteor test-packages ./packages/lesswrong --driver-package meteortesting:mocha --settings settings.json --port 3200 --full-app --once",
+    "generate": "graphql-codegen --config codegen.yml"
```

Comment:
IIUC this probably requires `graphql-codegen` to be installed globally, which won't happen automatically, but it could be rewritten to point into node_modules.

Reply:
Weirdly not. When a call to an npm package is part of package.json, it will automatically be executed with the npm cli-namespace. You can try it out.